cfgov/v1/management/commands/update_data_snapshot_values.py

```diff
@@ -0,0 +1,67 @@
+import ast
+
+from django.core.management.base import BaseCommand
+
+from v1.models.browse_page import BrowsePage
+from v1.tests.wagtail_pages.helpers import publish_changes
+
+
+class Command(BaseCommand):
+    help = 'Monthly updates to data snapshot values'
+
+    def add_arguments(self, parser):
+        """Adds all arguments to be processed."""
+        parser.add_argument(
```

Comment:
What about supporting some kind of file or `stdin` input, that way you can run the command using

```sh
$ cat input.json | manage.py command
```

or

```sh
$ manage.py command -i input.json
```

See for example [this suggestion](https://stackoverflow.com/q/7576525). I'm curious why it's better/easier to construct a complex string input on the command line vs. using an input file.